### ISSUE TITLE
Data Integration - Add truncate table to full table sync

### ISSUE THREAD

**Issue by dcahillm1** [mage-ai/mage-ai#4708]
**Is your feature request related to a problem? Please describe.**
I though that a Full table integration would truncate the table and then place the full table into the destination. However, the result is that the data is not truncated before.

**Describe the solution you'd like**
I think that having an extra option to either truncate the table or not should be enough. I understand that in some cases we just want to copy the whole table into a new partition and some other, just have one partition.

**Describe alternatives you've considered**
Running a block to truncate the table before running the integration pipeline.

**Comment by myrasangeetha** [comment]
Hi Team, I am developing Integration pipelines and facing same issue. Can i know the ETA on this above 
deployment please? Thanks!!

**Comment by myrasangeetha** [comment]
Thanks!

**Comment by myrasangeetha** [comment]
Hey Team, any update on the Truncate feature request?

**Comment by wangxiaoyou1993** [comment]
sorry about the delay. working on it now.

**Comment by sugimiyanto** [comment]
Hi there, any update on this?
thanks

**Comment by wangxiaoyou1993** [comment]
Should have this feature release in 1 or 2 weeks

**Comment by wangxiaoyou1993** [comment]
> Hi 😄 Just wanted to understand if this feature is released? Because in Jul 2024 was to be released but github ticket is still open. cc: [@wangxiaoyou1993](https://github.com/wangxiaoyou1993)

No, not yet. We're adding the data integration enhancements to Mage Pro at this moment.

**Comment by aabf** [comment]
@wangxiaoyou1993 Is there any date for open source release?
